ports/llvm/vcpkg.json

```
@@ -1,9 +1,19 @@
 {
   "name": "llvm",
-  "version-string": "12.0.1",
+  "version-string": "13.0.0",
```

```suggestion
  "version": "13.0.0",
```
This might be served by `version` here.

---

Fixed.